bokeh/models/tools.py

```diff
@@ -413,3 +413,13 @@ class HoverTool(Tool):
         the ``Rect`` and ``Quad`` glyphs are supported.
 
     """)
+
+    mode = Enum("point", "hline", "vline", help="""
+    Whether to consider hover pointer as point (x/y values), or a
+    span on h or v directions.
+    """)
+
+    hit_value_mode = Enum("snap_to_data", "glyph_center", "hit_interpolate",
```

Comment:
I think it has to be so, because you might want to interpolate any number of column values, actually, not just position. No idea now how this should be spelled or even implemented but I am confident this is not the right place for it :)